text/case.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { capitalizeWord, split } from "./_util.ts";
+
+/**
+ * Converts a input into camelCase
+ *
+ * @example
+ * ```ts
+ * import { toCamelCase } from "https://deno.land/std@$STD_VERSION/text/case.ts";
+ * import { assertEquals } from "https://deno.land/std@$STD_VERSION/testing/asserts.ts";
+ *
+ * assertEquals(toCamelCase("deno is awesome"), "denoIsAwesome");
```

Comment:
```suggestion
 * toCamelCase("deno is awesome"); // "denoIsAwesome"
```
This means we can remove the `assertEquals()` import too. Ditto for other instances.

Reply:
done